### PR TITLE
Add social media icons and community section

### PR DESCRIPTION
**Pull request by brifemu**
In this PR, I have merged all the changes made in the last month about social media stuff.

Open-Source-Project-Corona-Warn-App-Community

### CODE REVIEW

File: src/data/community_de.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
             "title": "Los geht's",
             "textblock": [
                 "Willkommen in der Corona-Warn-App-Community!",
-                "Wenn Du mehr darüber erfahren möchtest, wie Du bei uns mitmachen kannst, bist Du hier genau richtig. Nachfolgend findest Du Infos, wie Du bei uns mitarbeiten und zum Code und der Dokumentation der Corona-Warn-App beitragen kannst. Los geht's!"
+                "Wenn Du mehr darüber erfahren möchtest, wie Du bei uns mitmachen kannst, bist Du hier genau richtig. Nachfolgend findest Du Infos, wie Du bei uns mitarbeiten und zum Code und der Dokumentation der Corona-Warn-App beitragen kannst.",
+                "Du kannst uns auch auf Twitter und Instagram <a href='/de/community/#social-media'>folgen</a>, dich über die neusten Entwicklungen informieren und dich mit den anderen Followern unterhalten.",
```
Comment (MikeMcC399) [inline src/data/community_de.json:8]: The German text should be consistent about whether du / dich is written capitalized or not.

See the advice from Duden https://www.duden.de/sprachwissen/sprachratgeber/Gross-oder-Kleinschreibung-von-duDu-und-ihrIhr. 

I suggest following from the existing text and writing the pronouns capitalized.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Comment about du/Du. This applies also to "ihr" / "euch" as 2nd person plural. See Duden reference https://www.duden.de/sprachwissen/sprachratgeber/Gross-oder-Kleinschreibung-von-duDu-und-ihrIhr.

The equivalent of the German text "Du kannst uns auch auf Twitter und Instagram folgen, dich über die neusten Entwicklungen informieren und dich mit den anderen Followern unterhalten." is missing in the English version.

**Comment by dsarkar** [comment]
continues here https://github.com/corona-warn-app/cwa-website/pull/2284

File: src/data/community_de.json (hunk 1 above)
Comment (dsarkar) [reply src/data/community_de.json:8]: Thanks, we follow your suggestion and write the pronouns capitalized.

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for the review. Missing EN sentence was added.

**Pull request closed without merging**